### DISCUSSION TITLE
How do you rescale a physic object?

### DISCUSSION THREAD

**Discussion by Theonlyhamstertoh** [pmndrs/use-cannon#237] [Q&A]
In cannon.js, I can do something like this to change the scale of the object: 
```
bodyBody.removeShape(boxShape);
boxShape.halfExtents.set(1,2,3);
bodyBody.addShape(boxShape);
```
How would I go about doing this in use-cannon?

**Comment by drcmda** [comment]
i think its not implemented.

edit: not even in cannon :-( https://github.com/pmndrs/cannon-es/issues/102

re-making the entire object is not feasible imo. you want this to be as fast as possible for scaling animations. i think that should be a prop on the cannon.body object.
